incubator/kubeless/templates/ui-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-ui
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+    controller: {{ template "fullname" . }}-ui
+spec:
+  replicas: {{ .Values.controller.deployment.replicaCount }}
+  template:
+    metadata:
+      labels:
+        controller: {{ template "fullname" . }}-ui
+    spec:
+      containers:
+      - name: ui
+        image: "{{ .Values.ui.deployment.ui.image.repository }}:{{ .Values.ui.deployment.ui.image.tag }}"
+        imagePullPolicy: {{ .Values.ui.deployment.ui.image.pullPolicy }}
+        ports:
+        - containerPort: 3000
```

Comment:
I would suggesting naming the port here, probably as `http`.
Could we add health checks to the PodSpec?